meshtastic/interfaces/ble/lifecycle_service.py

```
@@ -0,0 +1,797 @@
+        BLELifecycleService._discard_invalidated_connected_client(
+            iface,
+            connected_client,
+            restore_address=restore_address,
+            restore_last_connection_request=restore_last_connection_request,
+        )
+        if is_closing:
+            raise iface.BLEError(ERROR_INTERFACE_CLOSING)
+        raise iface.BLEError(CONNECTION_ERROR_LOST_OWNERSHIP)
+
+    @staticmethod
+    def _verify_and_publish_connected(
```

Ruff (0.15.5)

[warning] Too many branches (19 > 12)

(PLR0912)